bootstrap/src/main/java/io/jenkins/jenkinsfile/runner/bootstrap/Bootstrap.java

```diff
@@ -338,12 +341,12 @@ public int run() throws Throwable {
         }
     }
 
-    public ClassLoader createJenkinsWarClassLoader() throws IOException, NoSuchMethodException, InvocationTargetException, IllegalAccessException {
-        return new ClassLoaderBuilder(new SideClassLoader(getPlatformClassloader()))
-                .collectJars(new File(warDir,"WEB-INF/lib"))
+    public ClassLoader createJenkinsWarClassLoader() throws PrivilegedActionException {
+        return AccessController.doPrivileged((PrivilegedExceptionAction<ClassLoader>) () -> new ClassLoaderBuilder(new SideClassLoader(getPlatformClassloader()))
```

Comment:
Jenkins is unlikely to work on advanced security policies . Should be fine though I will need to add Autotests for the resource loading

Reply:
This is pointless. See https://github.com/spotbugs/spotbugs/issues/1515